next/utils/utils.ts

```diff
@@ -52,3 +52,6 @@ const isServer = () => typeof window === 'undefined'
 export const isBrowser = () => !isServer()
 
 export const isProductionDeployment = () => process.env.NEXT_PUBLIC_IS_STAGING !== 'true'
+
+export const isObject = (value: any) =>
```

Comment:
I assume it's from here ? https://stackoverflow.com/questions/8511281/check-if-a-value-is-an-object-in-javascript if so (and in fact, even if it isn't) please add this link for context

```suggestion
// https://stackoverflow.com/questions/8511281/check-if-a-value-is-an-object-in-javascript
export const isObject = (value: any) =>
```